package.json

```diff
@@ -189,7 +189,7 @@
     "ts-jest": "^27.0.3",
     "typedoc": "^0.23.7",
```

Comment:
Do these need an update as well? I think in another pull request you updated them?

Reply:
There is a separate PR to update jest: https://github.com/inrupt/solid-client-js/pull/1802